src/client-side-metrics/exporter.ts

```diff
@@ -0,0 +1,460 @@
+  };
+  for (const scopeMetrics of exportArgs.scopeMetrics) {
+    for (const metric of scopeMetrics.metrics) {
+      const metricName = metric.descriptor.name;
+      if (isCounterMetric(metric)) {
+        for (const dataPoint of metric.dataPoints) {
+          // Extract attributes to labels based on their intended target (resource or metric)
+          const allAttributes = dataPoint.attributes;
+          const streaming = allAttributes.streamingOperation;
+          const metricLabels = Object.assign(
+            {
+              app_profile: allAttributes.appProfileId,
+              client_name: allAttributes.clientName,
+              method: allAttributes.methodName,
+              status:
+                (
+                  allAttributes as OnAttemptAttribute
+                ).attemptStatus?.toString() ??
+                (
+                  allAttributes as OnOperationAttribute
+                ).finalOperationStatus?.toString(),
+              client_uid: allAttributes.clientUid,
+            },
+            streaming ? {streaming} : null
+          );
+          const timeSeries = {
+            metric: {
+              type: metricName,
+              labels: metricLabels,
+            },
+            resource: {
+              type: exportArgs.resource._syncAttributes[
+                'monitored_resource.type'
+              ],
+              labels: resourceLabels,
+            },
+            valueType: 'INT64',
+            points: [
+              {
+                interval: {
+                  endTime: {
+                    seconds: dataPoint.endTime[0],
+                  },
+                  startTime: {
+                    seconds: dataPoint.startTime[0],
+                  },
+                },
+                value: {
+                  int64Value: dataPoint.value,
+                },
+              },
+            ],
+          };
+          timeSeriesArray.push(timeSeries);
+        }
+      } else {
+        for (const dataPoint of metric.dataPoints) {
+          // Extract attributes to labels based on their intended target (resource or metric)
+          const allAttributes = dataPoint.attributes;
+          const streaming = allAttributes.streamingOperation;
+          const metricLabels = Object.assign(
+            {
+              app_profile: allAttributes.appProfileId,
+              client_name: allAttributes.clientName,
+              method: allAttributes.methodName,
+              status:
+                (
+                  allAttributes as OnAttemptAttribute
+                ).attemptStatus?.toString() ??
+                (
+                  allAttributes as OnOperationAttribute
+                ).finalOperationStatus?.toString(),
+              client_uid: allAttributes.clientUid,
+            },
+            streaming ? {streaming} : null
+          );
+          const timeSeries = {
+            metric: {
+              type: metricName,
```

Comment:
This really ends up being the same thing as in [Python](https://github.com/googleapis/python-bigtable/pull/924/files#diff-142d623e0503afa707461f3bd80d8808a2e10232038c24d7d69ce1df6bd4dbceR180) even though a prefix isn't here. The prefix is just included when we define the open telemetry instruments.

Reply:
> I can't remember if you planned to support that or not

Not right now. [there isn't a need to support that right now](https://docs.google.com/document/d/1n_D1RBCYa44QTgtThEYgIQ3vLO0YuGeK8sAsilCwhIE/edit?disco=AAABc_QMooA) so I think we should leave that out to optimize for feature delivery time.

Reply:
We don't need to support this at this time as per the discussion [here](https://docs.google.com/document/d/1n_D1RBCYa44QTgtThEYgIQ3vLO0YuGeK8sAsilCwhIE/edit?disco=AAABc_QMooA).